sqlite-db/migrations/20220728115915_add_trading_pair.sql

```diff
@@ -0,0 +1,16 @@
+-- Introduce trading_pair field for all cfd tables.
+--
+-- Default to BTCUSD for all old already opened CFDs, as this is the only
+-- trading pair active for now.
+ALTER TABLE
+    cfds
+ADD
+    COLUMN trading_pair NOT NULL DEFAULT 'BtcUsd';
```

Comment:
I believe we are starting to introduce different terminologies 😬 
In the metrics I've used the term `symbol`. 

I've created an ADR for this so that we are all on the same page: https://github.com/itchysats/architecture/pull/35/files

Reply:
I'm aware that I was the one who introduced `trading symbol` 🙈

Reply:
all good, I was just following what was written in the ticket.... changed to `symbol`.